src/analyze/analyze.c

```diff
@@ -1248,6 +1339,30 @@ static int dot(int argc, char *argv[], void *userdata) {
 
         printf("digraph systemd {\n");
 
+        /* user configuration below ~/.config */
+        r = sd_path_home(SD_PATH_USER_CONFIGURATION, "systemd/analyze.conf.d", &user_config_dir);
+        if (r < 0)
+                return r;
+
+        r = config_parse_many_nulstr(PKGSYSCONFDIR "/analyze.conf",
+                                CONF_PATHS_NULSTR("systemd/analyze.conf.d"),
+                                "Color\0"
+                                "Style\0",
+                                config_item_table_lookup, items,
+                                CONFIG_PARSE_WARN, NULL);
+        if (r < 0)
+                return r;
+
+        /* allow to override by user */
+        r = config_parse_many_nulstr(NULL,
+                                user_config_dir,
+                                "Color\0"
+                                "Style\0",
+                                config_item_table_lookup, items,
+                                CONFIG_PARSE_WARN, NULL);
```

Comment:
this should be folded into the above, i.e. use an strv instead, and put it together first, then call config_parse_many() only once

Reply:
done